### PR TITLE
Fix position of icon and highlighted text

### PR DESCRIPTION
**Pull request by JiwoonKim**
**Closes Issue** #36 
- vertically center the icon with the tooltip text
- replace blanklines Typography component with div element
- add more padding to highlighted text and fix font-size and position if necessary

**Applied Changes**
- replaced blanklines Typography component with div element
- refactored `<Tooltip />` into its own separate component
- vertically centered the icon for the tooltip text
- fixed height or rows and lines + added padding for highlighted text (for `types` in left panel)

**Result Screenshot**

### CODE REVIEW

File: src/components/SchemaTable/index.jsx [hunk 1]
```diff
@@ -36,12 +36,19 @@ const useStyles = makeStyles(theme => ({
   keywordColumn: {},
   descriptionColumn: {},
   line: {
+    display: 'flex',
+    alignItems: 'center',
```
Comment (helfi92) [inline src/components/SchemaTable/index.jsx]: It's centered on Chrome but on Firefox, it looks a bit off:

Screen Shot 2019-12-27 at 8 59 14 AM

Comment (JiwoonKim) [reply src/components/SchemaTable/index.jsx]: Applied changes! Hope this is better 😃 

| Chrome |  Firefox |
|:---|:---|
|  |  |

**Comment by JiwoonKim** [comment]
> Weird, the issue is still there on Firefox. This is how it looks like:
> Are you able to replicate?

Hmmm... that's strange.
The icon seems to be placed in align with the text in my Firefox browser.

**Comment by JiwoonKim** [comment]
Added some structure and style changes to the `Tooltip` component!
(also added a red line border to make it easier to compare icon position with the text, 
will remove it when this PR is approved 😄 )

This is what the changes applied look like in Firefox browser:


Does it align correctly now?

> Also, there seems to be a console warning about a unique key. Just wanted to point this out. Feel free to fix this in a different PR.

I've also made some changes to fix this issue as well. There should be no console warning anymore :)

File: src/components/SchemaTable/index.jsx [hunk 2]
```diff
@@ -51,16 +51,24 @@ const useStyles = makeStyles(theme => ({
...
    * Highlighting the type for the schema or sub-schema displayed
    * in the left panel of the schema table component.
    */
   code: {
     backgroundColor: theme.palette.grey[300],
+    padding: `0 ${theme.spacing(0.5)}px`,
+  },
+  tooltip: {
+    fontSize: `${theme.typography.subtitle2.fontSize}`,
+    lineHeight: 1,
```
Comment (helfi92) [inline src/components/SchemaTable/index.jsx]: The `lineHeight` seems to have been the fix. Awesome work tracking this down! It was a tricky one.

File: src/components/Tooltip/index.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  return (
+    <MuiTooltip title={createTooltipTitle(keyword)} arrow>
+      <div className={classes.line}>
+        <Typography
+          component="div"
+          variant="subtitle2"
+          className={classes.tooltip}>
+          {`${keyword}: `}
+        </Typography>
```
Comment (helfi92) [inline src/components/Tooltip/index.jsx]: When we're showing the icon, what do you think about removing the column and adding some margin (e.g., 4px instead)?

_Example: removing the column and adding some margin on propertyNames_

Screen Shot 2019-12-31 at 4 42 36 PM

Comment (JiwoonKim) [reply src/components/Tooltip/index.jsx]: Great idea! Removed the colon from Tooltip's text.
It makes the UI look more cleaner 👍 

image

**Pull request merged**